pkgs/applications/editors/gnome-latex/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, wrapGAppsHook
+, tepl, amtk, gnome3, glib, pkgconfig, intltool, itstool, libxml2 }:
+let
+  version = "3.28.1";
+  pname = "gnome-latex";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
```

Comment:
Sorry about misinformation, we now have `stdenv.lib.versions.majorMinor` which should be preferred.